rules/brainstorming-idea-champion/rule.md

```diff
@@ -0,0 +1,38 @@
+---
+type: rule
+title: Do you have a brainstorming idea champion?
+uri: brainstorming-idea-champion
+authors:
+  - title: Seth Daily
+    url: https://www.ssw.com.au/people/seth-daily
+  - title: Piers Sinclair
+    url: https://www.ssw.com.au/people/piers-sinclair
+created: 2024-04-02T13:42:55.753Z
+guid: 0034f3e2-a3b5-4527-8r3d-45b27f2e4a1f
+---
+
+During a Brainstorming day, many great ideas are generated, discussed, and refined. But what happens to these ideas once the day is over? Without a dedicated individual to drive an idea forward, it risks being forgotten or losing momentum. This is where the Idea Champion comes in.
+
+<!--endintro-->
+
+The Idea Champion should be someone who believes in the idea's potential.
+
+## The Idea Champion's responsibilities
+
+* **Take accountability** for the idea from the end of the brainstorming day until a Scrum Team has been assigned and development has started
```

Comment:
Ideas don't always need to turn into Scrum teams - sometimes we decide they are not actioned at all which is a valid outcome.

```suggestion
* **Take accountability** for the idea from the end of the brainstorming day until a Scrum Team has been assigned and development has started or the idea is parked
```

Reply:
I added this comment before reading the last line of the rule, but i still think its valid